scripts/bootloader/provision.py

```diff
@@ -55,6 +55,9 @@ def parse_args():
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
     parser.add_argument("--generated-conf-file", required=True, help="Generated conf file.")
+    parser.add_argument("--s0-addr", type=int, required=False, help="Set address of s0 explicit")
```

Comment:
In next three lines:
explicit -> explicitly

Reply:
fixed (not pushed)